odl/contrib/fom/supervised.py

```diff
@@ -519,11 +571,19 @@ def smoothen(img):
     sigma2_sq = smoothen(ground_truth * ground_truth) - mu2_sq
     sigma12 = smoothen(data * ground_truth) - mu1_mu2
 
-    nom = (2 * mu1_mu2 + C1) * (2 * sigma12 + C2)
+    num = (2 * mu1_mu2 + C1) * (2 * sigma12 + C2)
     denom = (mu1_sq + mu2_sq + C1) * (sigma1_sq + sigma2_sq + C2)
-    pointwise_ssim = nom / denom
+    pointwise_ssim = num / denom
+
+    result = np.mean(pointwise_ssim)
+
+    if normalized:
+        result = (result + 1.0) / 2.0
+
+    if force_lower_is_better:
+        result = -result
```

Comment:
This looks like it would return the wrong value if both `force_lower_is_better=True` and `normalize=True`. Reordering these if statements fixes that.